### ISSUE TITLE
Auto-collapsed sections instead of very long nonoverseeable pages

### ISSUE THREAD

**Issue by prototyperspective** [WDscholia/scholia#1727]
Could you make all sections be collapsed so that it's not a gigantic nonoverseeable page but a quick-loading small clear one (and with no information-overload and better to browse, especially when browsing many pages)?

Having very long pages is confusingly complex / unclear and hard to browse. Example pages (to showcase Scholia) could still have all section uncollapsed (for example via query strings).

**Comment by prototyperspective** [comment]
This would be especially useful for mobile. It would also be useful to have a sidepanel of the TOC so you can easily switch between sections, especially on mobile.

**Comment by fnielsen** [comment]
Yes. With my lowcost smartphone viewing Scholia is a bit of a challenge. The graphs are not well-suited to the small scale. I note that Wikipedia as a site focused on Mobile service m.wikipedia.org. I suppose it would be quite a task to make an "extra" site.
